homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -50,6 +50,9 @@ def __init__(self, hass: HomeAssistant, *, entry: ConfigEntry) -> None:
             update_interval=timedelta(seconds=SCAN_INTERVALS[entry.data[CONF_REGION]]),
         )
 
+        # Default to false on init so _async_update_data logic works
+        self.last_update_success = False
```

Comment:
I am not 100% sure if there aren't any side effects of this as the base class sets this to true, and presumably that is for a reason. I can look more deeply when I'm back at a real computer.

In any case I wondered if setting to `None` here would achieve the same thing and be a bit more obvious that the last update was non-existent rather than a failure.

Reply:
I think `None` would wrong in this case as `last_update_success` is defined as non-optional `bool` (cf. [update_coordinator.py#L107](https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/update_coordinator.py#L107)).